### ISSUE TITLE
[Bug] v.4.9.82 CSS break nearly all websites!

### ISSUE THREAD

**Issue by Dragodraki** [darkreader/darkreader#12580]
### Prerequisites

- [X] I [searched for any existing report](https://github.com/darkreader/darkreader/issues?q=is%3Aissue) about this bug to avoid opening a duplicate.
- [X] I can reproduce this bug in a new, unmodified web browser profile with Dark Reader installed as the only extension.
- [X] I understand I need to use the [Broken Website Report template](https://github.com/darkreader/darkreader/issues/new?assignees=&labels=Broken+Website&template=broken-website-report.yml&title=%5BBroken+Website%5D+Replace+with+title) if this bug I am reporting occurs on a single website.

### Bug Description

Using your latest version v.4.9.82 the CSS with Firefox 115.9.1esr on Windows 11. Your latest update rather seems like it destroys nearly all website styles. Where the background color was once black it is white now, so the text does not highlight from it (missing contrast) anymore. The "Dynamic" mode is affected only, but within it nearly all webpages.
One example here (though you'll find probably lots of more): 
https://jrsoftware.org/ishelp/index.php?topic=consts

Please make it like it was before again. In the meantime I rollback to v.4.9.81 and block addon updates (really cannot work with this broken "Dynamic" style).

### Website Address

No clue what it has to do with a bug report

### Steps To Reproduce

1. Install DarkReader v.4.9.82
2. Set mode to "Dynamic"
3. Visit some webpages - they were black till v.4.9.81 and now they are white again with fade text, so contrast is really bad

### Expected Behavior

Dark background color

### Actual Behavior

Fair background color

### Operating System

Windows 11 Pro 64-Bit

### Web Browser name and version

Firefox 115.9.1esr 64-Bit

### Dark Reader version

v.4.9.82

**Comment by Dragodraki** [comment]
No, besides the standard "undefined: undefined" error at the beginning (that appears everywhere and with/without dark reader addon installed) no error in console.

**Comment by alexanderby** [comment]
The website you mentioned has a problem with iframes. I submitted a fix, it should be available soon.

Could you please share any other URLs?

**Comment by Dragodraki** [comment]
Fixed in version 4.9.83, thanks you :)
